tests/functional/api.php

```diff
@@ -0,0 +1,60 @@
+<?php
+define('INSTALL_ROOT',  realpath(__DIR__ . '/../../') . '/');
+
+// We always work with UTF8 encoding
+mb_internal_encoding('UTF-8');
+
+// Make sure we have a timezone set
+date_default_timezone_set('Europe/Paris');
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+// Launch PHP dev server in the background
+chdir(INSTALL_ROOT);
+exec('php -S 0.0.0.0:8083 > /dev/null 2>&1 & echo $!', $output);
+
+// We will need the pid to kill it, beware, this is the pid of the bash process started above
```

Comment:
We are not launching a bash script but directly a php script, update the comment